mypy.ini

```diff
@@ -4,15 +4,19 @@ files = torchvision
 show_error_codes = True
 pretty = True
 
-;[mypy-torchvision.datasets.*]
+[mypy-torchvision.io.*]
 
-;ignore_errors = True
+ignore_errors = True
 
-[mypy-torchvision.io.*]
+[mypy-torchvision.models.detection.*]
```

Comment:
In total `mypy` found 86 existing errors in `detection`, `densenet`, and `quantization`. I'm ignoring them here to get a clean run and will fix them in separate follow-up PRs.